docs/release-notes.rst

```diff
@@ -6,6 +6,71 @@
  Release Notes
 ###############
 
+**************
+ Version 0.36
+**************
+
+Version 0.36.0
+==============
+
+**Release Date:** August 22, 2024
+
+**New Features**
+
+-  WebUI: In the enterprise edition of Determined, when RBAC is enabled, allow only Viewer, Editor,
+   GenAI, and Workspace Admin roles to view resource quotas for each workspace in the WebUI. When
+   RBAC is not enabled, any user can view resource quotas.
+
+-  RBAC: Add a pre-canned role called ``EditorProjectRestricted`` that supersedes the ``Viewer``
+   role and precedes the ``Editor`` role.
+
+   -  Like the ``Editor`` role, the ``EditorProjectRestricted`` role grants the permissions to
+      create, edit, or delete experiments and NSC (Notebook, Shell or Command) type workloads within
+      its scope. However, the ``EditorProjectRestricted`` role lacks the permissions to create or
```

Comment:
I would be more direct here by what we mean within its scope --> don't we mean "within the project"? I would just overly clarify the exact behavior is reading this RBAC stuff for the first time can lead to confusion for users

Reply:
This is not within the project, this is within whatever the Editor is scoped to which could be one or more workspaces or globally so within its scope is right. Maybe we could add in brackets something like ...within its scope (workspace(s) or global)

Reply:
Its probably redundant but maybe we can be explicit here and say "read, create, edit or delete experiments...". Don't think it's a big deal since I dont think its possible to have a situation where you can edit but not read

Reply:
i agree the rbac roles are very difficult to decipher in sentence format, particularly when we start saying this role supercedes this role, this role precedes this role.... i have a docs ticket to better describe these and help users (and us) visualize them. we can even make a flowchart pictogram if needed.

Reply:
the suggested change has been added